docs/content/Flex.md

```diff
@@ -2,9 +2,9 @@
 title: Flex
 ---
 
-Flex is a component that will allow you to use flexbox properties.
+The `Flex` component behaves the same as the `Box` component except that it has `display: flex` set by default.
 
-*Previously, `Flex.Item` was used for flex item specific properties - we've added all properties to the `Flex` component, but are keeping Flex.Item around for backwards compatibility.*
+*Previously, `Flex.Item` was used for flex item specific properties - we've added all properties to the `Box` and `Flex` components, but are keeping `Flex.Item` around for backwards compatibility.*
```

Comment:
I wonder if we should go ahead and remove `Flex.Item` now since it's been a while 🤔

Reply:
I marked it as deprecated in the docs, but since we're making breaking changes I think we should just remove it. 👍